lib/image_optim/worker/optipng.rb

```diff
@@ -42,6 +42,10 @@ def optimize(src, dst)
         end
         execute(:optipng, *args) && optimized?(src, dst)
       end
+
+      def optimized?(src, dst)
+        (interlace && dst.size?) || super
```

Comment:
If `interlace` is `true`, running `super` is not needed, so `interlace ? dst.size? : super` would be clearer.